src/app/common/inbox-linshare-api-client.service.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+angular.module('linagora.esn.unifiedinbox.linshare')
+  .factory('inboxLinshareApiClient', inboxLinshareApiClient);
```

Comment:
This will cause an issue in Webpack production mode. It should be `inboxLinshareApiClientFactory`